downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

```diff
@@ -68,8 +103,67 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
+| `getRestFieldType`                                                       | `io.ballerina.runtime.api.types.RecordType`        |
+| `getReferredType`                                                        | `io.ballerina.runtime.api.types.ReferenceType`     |
+| `getConstrainedType`                                                     | `io.ballerina.runtime.api.types.StreamType`        |
+| `getCompletionType`                                                      | `io.ballerina.runtime.api.types.StreamType`        |
+| `getConstrainedType`                                                     | `io.ballerina.runtime.api.types.TableType`         |
+| `getTupleTypes`                                                          | `io.ballerina.runtime.api.types.TupleType`         |
+| `getRestType`                                                            | `io.ballerina.runtime.api.types.TupleType`         |
+| `getConstraint`                                                          | `io.ballerina.runtime.api.types.TypedescType`      |
+| `getMemberTypes`                                                         | `io.ballerina.runtime.api.types.UnionType`         |
+| `getOriginalMemberTypes`                                                 | `io.ballerina.runtime.api.types.UnionType`         |
+| `getElementType`                                                         | `io.ballerina.runtime.api.values.BArray`           |
+| `getConstraintType`                                                      | `io.ballerina.runtime.api.values.BStream`          |
+| `getCompletionType`                                                      | `io.ballerina.runtime.api.values.BStream`          |
+| `getKeyType`                                                             | `io.ballerina.runtime.api.values.BTable`           |
+| `getDescribingType`                                                      | `io.ballerina.runtime.api.values.BTypedesc`        |
+| `getType`                                                                | `io.ballerina.runtime.api.values.BValue`           |
+| `getType`                                                                | `io.ballerina.runtime.api.utils.TypeUtils`         |
+
+For example, if the type-reference types are defined in the following way,
+
+```ballerina
+type ReadonlyIntArray readonly & int[];
+
+ReadonlyIntArray array = [1, 2, 3, 4];
+```
+the results of the runtime API calls will be as follows.
+
+| **Runtime API call**                                     | **Result**                                                      |
+|----------------------------------------------------------|-----------------------------------------------------------------|
+| `array.getType()`                                        | This will return an `IntersectionType` insted of `ArrayType`.   |
+| `getDescribingType()` on `ReadonlyIntArray` type         | This will return an `IntersectionType` insted of `ArrayType`.   |
```

Comment:
```suggestion
| `array.getType()`                                        | This will return an `IntersectionType` instead of `ArrayType`.   |
| `getDescribingType()` on `ReadonlyIntArray` type         | This will return an `IntersectionType` instead of `ArrayType`.   |
```